### DISCUSSION TITLE
[Question]Can I skip flutter doctor phase?

### DISCUSSION THREAD

**Discussion by DFNJKD-98** [flet-dev/flet#4359] [Q&A] [answered]
### Question

Due to some reasons, my country cannot freely access GitHub, so Flutter Doctor always reports errors. This hinders me from using Fleet to build Windows applications. Can I skip the Flutter Doctor phase using the 'flex build windows' command? I'm not sure if this is necessary.

### Error message

```bash
As shown below, the Flutter Doctor command may encounter some issues, but building a Windows application directly using Flutter is not a problem. I can use Flutter normally in VSCode.
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, 3.22.1, on Microsoft Windows [version 1.0.19045.4046], local zh CN)
[√] Windows Version (Installed version of Windows is version 10 or higher)
[√] Chrome - develop for the web
[√] Visual Studio - develop Windows apps (Visual Studio Community 2022 17.7.3)
[!] Flutter IDE Support (No supported IDEs installed)
[√] Connected device (3 available)
[!] Network resources
X An HTTP error occurred while checking " https://github.com/ ": Connection closed before full header was received

! Doctor found issues in 2 categories.
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
I created an issue: https://github.com/flet-dev/flet/issues/4369
Watch it for updates.
